modules/data-store/test/data-store.test.js

```diff
@@ -0,0 +1,215 @@
+const DataStore = require("../src/data-store.js");
+const Entity = require("../src/entity");
+
+
+describe("DataStore.addEntity", () => {
+    test("entity is stored into an object", () => {
+        const dataStore = new DataStore();
+        const entity = new Entity({ id: "entity-0" });
+        dataStore.addEntity(entity, "/tata");
+        const object = {
+            "/tata": [entity],
+        };
+        const object2 = {
+            "/toto": [entity],
+        };
+        expect(dataStore.entitiesByPath).toEqual(object);
+        expect(dataStore.entitiesByPath).not.toEqual(object2);
+    });
+    test("many entities in same path", () => {
```

Comment:
Can you add blank line between tests ? :)

Reply:
Done